include/aikido/util/yaml_utils.hpp

```diff
@@ -0,0 +1,202 @@
+#ifndef AIKIDO_UTIL_YAMLUTILS_HPP_
+#define AIKIDO_UTIL_YAMLUTILS_HPP_
+
+#include <functional>
+#include <mutex>
+#include <vector>
+#include <iostream>
+#include <boost/format.hpp>
+#include <boost/make_shared.hpp>
+#include <Eigen/Dense>
+#include <yaml-cpp/yaml.h>
+
+
+namespace aikido {
+namespace util {
+
+template <class _Scalar,
+          int _Rows, int _Cols, int _Options, int _MaxRows, int _MaxCols>
+inline void deserialize(
```

Comment:
I believe we don't use this function out of `template <...> struct convert`. Could I just move this into it as you said?